js/iframe-coordinator/libs/client.ts

```diff
@@ -104,5 +144,4 @@ class Client {
   }
 }
 
-const client = new Client();
-export default client;
\ No newline at end of file
+export { Client };
```

Comment:
Exporting the constructor now.

Reply:
Btw, I really am disliking the name "Client".  I purpose we may want to name it something else.  Maybe something similar to Parcel or Fragment like mosaic/single-spa uses for their micro-frontends?

Reply:
I used to have a different set of names for the parent app and the embedded app, something like "coordinator" and "component" but after taking an informal survey (me walking around the office asking folks), it seemed like the terms "host" and "client" for the parent app and the iframe were the things that people found easiest to understand quickly which thing was which in the system.  Do you think Client is too generic? It's certainly an overloaded term and I could see how it could be confusing in the context of an app that might have clients for other things.

Reply:
Yea, I think its too generic and overloaded.  I just feel we could give it something more descriptive to its purpose.